tests/ut_common_processmix.py

```diff
@@ -0,0 +1,15 @@
+"""
+The unit test checks if process_mix function works fine.
+"""
```

Comment:
Let's get SeattleTestbed/seattlelib_v2#161 under control before adding this one. We want to make sure that `repypp.py` works correctly before attempting to use it.